runtime/Makefile

```diff
@@ -9,13 +9,10 @@ QIR_STDLIB_DIR?=$(MK_DIR)/qir-stdlib/target/release/
 QIR_STDLIB_INCLUDES_DIR?=$(MK_DIR)/qir-stdlib/target/release/build/include/
 CODE_COVERAGE?=OFF
 BUILD_TYPE?=Release
-ENABLE_KOKKOS?=OFF
-ifeq ($(ENABLE_KOKKOS),OFF)
-	LIGHTNING_GIT_TAG_VALUE?=latest_release
-else
-	# TODO: Update this to latest_release for the v0.30.0 release
-	LIGHTNING_GIT_TAG_VALUE?=main
-endif
+ENABLE_WARNINGS?=ON
+ENABLE_OPENMP?=ON
```

Comment:
Yep, they would let us to have control over the use of OpenMP kernels in both Lightning and Lightning-Kokkos as well as treating compiler warnings as errors.

Reply:
Right that's what the flags do (which is nice!), but I was curious if there was specific situation that made you realize this would be nice to have - no change needed though 🙂

Reply:
Lightning is built with OpenMP kernels for the use in PennyLane and we used to build Lightning device using these kernels. To highlight the point that we're building Lightning (and now Lightning-Kokkos) with OpenMP, I've added this macro in the Makefile so that's clear what're all flags we're using to build devices and one can turn it OFF if needed.

Reply:
Thanks :)